app/javascript/src/components/Dashboard/Notes/Table.jsx

```diff
@@ -1,27 +1,22 @@
 import React, { useState } from "react";
 
-import { Table as NeetoUITable } from "neetoui";
-
-import { NOTES_TABLE_COLUMN_DATA } from "./constants";
+import NoteCard from "./NoteCard";
 import EditNotePane from "./Pane/Edit";
 
-const Table = ({ setSelectedNoteIds, notes = [], fetchNotes }) => {
+const Table = ({ notes = [], fetchNotes }) => {
   const [showEditNote, setShowEditNote] = useState(false);
-  const [selectedNote, setSelectedNote] = useState({});
-
+  const [selectedNote] = useState({});
```

Comment:
This is an anti-pattern. Why have you declared a state without a setter? If we don't need to set this state and rerender UI, then what's the point of making this a state?

Reply:
Having the unused variable (`setSelectedNote`) was causing a pre-commit hook to fail so I just removed it. But as you pointed out, this isn't how it's done and I should've just made it a simple variable. Would keep it in mind.